ddl/backfilling_scheduler.go

```diff
@@ -439,30 +472,24 @@ func (w *addIndexIngestWorker) HandleTask(rs idxRecResult) {
 		w.resultCh <- result
 		return
 	}
-	count, nextKey, err := w.WriteLocal(&rs)
+	count, err := w.WriteLocal(&rs)
 	if err != nil {
 		result.err = err
 		w.resultCh <- result
 		return
 	}
 	if count == 0 {
 		logutil.BgLogger().Info("[ddl-ingest] finish a cop-request task", zap.Int("id", rs.id))
-		if bc, ok := ingest.LitBackCtxMgr.Load(w.jobID); ok {
-			err := bc.Flush(w.index.Meta().ID)
-			if err != nil {
-				result.err = err
-				w.resultCh <- result
-			}
-		}
 		return
 	}
-	result.scanCount = count
-	result.addedCount = count
+	cnt, nextKey := w.checkpointMgr.Status()
+	result.totalCount = cnt
```

Comment:
Whether line 486 should be put after line 492? the count should be added first then  set to result.totalCount?

Reply:
The effect is exactly the same since no one reads the result until it is sent back to the channel(`w.resultCh`).